config/initializers/rollbar.rb

```diff
@@ -6,8 +6,13 @@
   config.access_token = ENV['ROLLBAR_ACCESS_TOKEN']
   config.environment = ENV['ROLLBAR_ENV']
 
-  # Here we'll disable Rollbar when not in 'production':
-  config.enabled = false unless Rails.env.production?
+  # Rollbar is only enabled in 'production' outside of rake tasks
+  if !Rails.env.production? || /rake/ =~ File.split($PROGRAM_NAME).last
+    config.enabled = false
+  end
+
+  # Note: 'rake rollbar:test' will not properly test the setup since
+  #       Rollbar is now disabled in rake tasks.
```

Comment:
Small tweak: Could you move this just after Line 9's comment? That way it's easily seen when reading top-to-bottom for what Line 10 particularly does.

Reply:
Done and done!